### ISSUE TITLE
[Question] Validating Tailwind classes inside object values accessed by key

### ISSUE THREAD

**Issue by lukasvice** [schoero/eslint-plugin-readable-tailwind#211]
Hi 👋,

First of all, thank you for the great work on this plugin! I’ve been able to get very good results by tweaking the `callees` setting for `classNames`.

My current `callees` configuration looks like this:

```ts
[
  "classNames",
  [
    { "match": "strings" },
    {
      "match": "objectKeys",
      "pathPattern": ".*"
    }
  ]
]
```

This works perfectly for most cases, e.g.:

```ts
classNames('flex', ['flex-row', 'gap-2'], { 'flex-wrap': true });
```

However, I’m struggling to validate Tailwind classes in a slightly different pattern — where the values of an object are class strings, and I access them by a key:

```ts
classNames(
  {
    ['option1']: 'px-3',
    ['option2']: 'px-4',
    ['option3']: 'px-6',
  }[selectedOption]
);
```

I tried enabling `"match": "objectValues"`, but it didn’t seem to have any effect.

**My question:**
Is it currently possible to validate class strings in object values that are accessed dynamically like this?
If not, would you recommend a different approach or code structure to achieve this?

Thanks in advance for your guidance! 🙏

**Comment by lukasvice** [comment]
Wow, thank you so much! 🚀
